pkgs/by-name/cc/cctools/meson.build

```diff
@@ -553,7 +539,7 @@ install_man('man/strings.1')
 strip = executable(
     f'@target_prefix@strip',
     c_args : ['-DTRIE_SUPPORT'],
-    dependencies : [libcodedirectory, libprunetrie],
+    dependencies : [libcxx, libcodedirectory, libprunetrie],
```

Comment:
weird. I only see strip needed to link with lib stuff
https://github.com/apple-oss-distributions/cctools/blob/7a5450708479bbff61527d5e0c32a3f7b7e4c1d0/cctools.xcodeproj/project.pbxproj#L2221-L2228

Reply:
Yeah, so it turns out that the Xcode build system is even more baroque than you can imagine; there’s [another file](https://github.com/apple-oss-distributions/cctools/blob/7a5450708479bbff61527d5e0c32a3f7b7e4c1d0/xcode/strip.xcconfig).